velocity-tools-view/src/main/java/org/apache/velocity/tools/view/ServletUtils.java

```diff
@@ -80,9 +80,9 @@ public static String getPath(HttpServletRequest request)
         // will return the original (wrong) URI requested.  The following special
         // attribute holds the correct path.  See section 8.3 of the Servlet
         // 2.3 specification.
-        String path = (String)request.getAttribute("javax.servlet.include.servlet_path");
+        String path = (String)request.getAttribute("jakarta.servlet.include.servlet_path");
```

Comment:
There is https://tomcat.apache.org/tomcat-9.0-doc/servletapi/javax/servlet/RequestDispatcher.html#INCLUDE_SERVLET_PATH. We should prefer over literals.